catroid/src/main/res/values/strings.xml

```diff
@@ -157,84 +157,84 @@
+    <string name="preference_description_phiro_bricks">Allow the app to control RobotixEdu\'s Phiro robots</string>
+    <string name="preference_title_enable_arduino_bricks">Arduino extension</string>
+     <string name="preference_description_arduino_bricks">Allow the app to control Arduino boards</string>
+    <string name="preference_title_enable_embroidery_bricks">Embroidery extension</string>
     <string name="preference_description_embroidery_bricks">Create patterns for stitching machines</string>
-    <string name="preference_title_cast_feature_globally_enabled">Enable cast features</string>
+    <string name="preference_title_cast_feature_globally_enabled">Chromecast extension</string>
     <string name="preference_description_cast_feature_globally_enabled">Show cast button and enable cast projects</string>
 
-    <string name="preference_title_enable_nfc_bricks">Near Field Communication</string>
-    <string name="preference_description_nfc_bricks">Allow the app to use NFC tags in programs</string>
+    <string name="preference_title_enable_nfc_bricks">Near Field Communication extension</string>
+    <string name="preference_description_nfc_bricks">Allow the app to use NFC tags in projects</string>
     <string name="preference_title_enable_crash_reports">Sending anonymous crash reports</string>
     <string name="preference_description_crash_reports">Allow the app to send crash reports automatically</string>
     <!-- -->
-    <string name="preference_description_arduino_bricks">Allow the app to control Arduino boards</string>
-    <string name="preference_title_enable_raspi_bricks">Raspberry Pi Bricks</string>
+    <string name="preference_title_enable_raspi_bricks">Raspberry Pi extension</string>
     <string name="preference_description_raspi_bricks">Allow the app to control Raspberry Pi computers</string>
-    <string name="preference_raspi_enable_bricks">Enable Raspberry Pi Bricks</string>
-    <string name="preference_raspi_help">Help / Getting Started</string>
+    <string name="preference_raspi_enable_bricks">Enable bricks and sensors</string>
+    <string name="preference_raspi_help">Help / getting started</string>
     <string name="preference_raspi_help_summary">Get help on how to configure the connection settings</string>
-    <string name="preference_raspi_host">Raspberry Pi IP-/Host-Address</string>
-    <string name="preference_raspi_port">Raspberry Pi Port</string>
+    <string name="preference_raspi_host">Raspberry Pi IP-/host-address</string>
+    <string name="preference_raspi_port">Raspberry Pi port</string>
     <!--  -->
 
     <!-- Scratch Converter activity -->
     <string name="beta">BETA</string>
-    <string name="error_scratch_program_not_accessible_any_more">Whoops! The program seems to have disappeared
+    <string name="error_scratch_program_not_accessible_any_more">Whoops! The project seems to have disappeared
```

Comment:
General rule: "project" makes more sense than "program", and is easier to understand for novices. It also emphasizes the "tinkering" aspect (a project focuses on the "doing" stuff instead of the "product"), or the working in teams on a project. For more on this, also see http://web.media.mit.edu/~mres/papers/IJCCI-seeds-seymour-sowed.pdf